test/application_system_test_case.rb

```diff
@@ -104,6 +104,12 @@ def fail_on_js_error(error)
 
   private
 
+  def swallow_js_errors(reason: "I know what I am doing")
```

Comment:
I like this helper method. My only thought is that while “swallow” is a fairly colloquial way to refer to this kind of behavior, “ignore” is a little more clear and may be more likely to be understood by a wider range of developers.

```suggestion
  def ignore_js_errors(reason: "I know what I am doing")
```

Reply:
I updated the commit and PR to have the better naming. Good call on that!